recipes/tesseract/all/conanfile.py

```diff
@@ -59,13 +59,13 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("leptonica/1.82.0")
+        self.requires("leptonica/1.83.0", transitive_headers=True)
         # libarchive is required for 4.x so default value is true
         if self.options.get_safe("with_libarchive", default=True):
             self.requires("libarchive/3.6.2")
         # libcurl is not required for 4.x
         if self.options.get_safe("with_libcurl", default=False):
-            self.requires("libcurl/7.86.0")
+            self.requires("libcurl/8.0.1")
```

Comment:
Did you test that the major version bump works properly? They are always scary :$